etc/config-template.toml

```diff
@@ -1086,6 +1086,16 @@
 ## When it's not set, 15% of available system memory will be used.
 # memory-limit = "1GB"
 
+## Version of raft logs in Raft Engine.
```

Comment:
Keep the description consistent with https://github.com/tikv/raft-engine/blob/master/src/config.rs. But add an additional line to explain V2 is only readable by tikv 6.3 and above.